### PR TITLE
chore: add script to compute code samples coverage

### PR DESCRIPTION
**Pull request by shobsi**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: scripts/get_code_sample_coverage.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    bigframes.operations.strings.StringMethods,
+    bigframes.operations.datetimes.DatetimeMethods,
+    bigframes.operations.structs.StructAccessor,
+]
+
+ML_MODULE_NAMES = [
+    "cluster",
+    "compose",
+    "decomposition",
+    "linear_model",
```
Comment (GarrettWu) [inline scripts/get_code_sample_coverage.py:39]: + ensemble

File: scripts/get_code_sample_coverage.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+import inspect
+import sys
+from typing import Dict, List
+
+import bigframes
+import bigframes.pandas as bpd
+
+PRESENT = "present"
+NOT_PRESENT = "not_present"
+
```
Comment (GarrettWu) [inline scripts/get_code_sample_coverage.py:26]: Why not include I/O, remote_function etc.

Comment (shobsi) [reply scripts/get_code_sample_coverage.py:26]: Included Session class which contain I/O and remote function methods.

File: scripts/get_code_sample_coverage.py (hunk 1 above)
Comment (shobsi) [reply scripts/get_code_sample_coverage.py:42]: Included in the latest patch.

**Review by GarrettWu — APPROVED** [review]

**Pull request merged**